spec/services/data_export_service_spec.rb

```diff
@@ -47,7 +47,8 @@
         data = data_export.reload.data
         headers = data[0]
         status_index = headers.index("Estado")
-        expect(data[1][status_index]).to eq "Activo"
+        # use second row because second row of standard loan data export has question ids
+        expect(data[2][status_index]).to eq "Activo"
       end
```

Comment:
this particularity coming thru in the service spec is making the 'standard loan data export' not feel very 'standard' to me. I think it may be easier to manage things in the future if we break the question data out into a separate data export, which can be another issue next sprint. I put more about that in the PR comment. What do you think @meltheadorable ?